### DISCUSSION TITLE
无法登录lantern

### DISCUSSION THREAD

**Discussion by sparkinpan** [getlantern/lantern-forum-cn#762] [General]
os版本win11，蓝灯稳定版，安装后出现localhost拒绝访问。如何解决。
另一个问题：os版本win11，蓝灯beta版，无法通过邮件或授权码给新电脑授权

**Comment by MaxMKMax** [comment]
@sparkinpan 您好. 请您提供我支付单据和账号邮箱. 我帮您拿授权码.
